scripts/test-check-packages.sh

```diff
@@ -10,7 +10,7 @@ cleanup() {
 
   # Dump kubectl details
   kubectl describe pods --all-namespaces > build/kubectl-dump.txt
-  kubectl logs -l app=kind-fleet-agent-clusterscope -n kube-system >> build/kubectl-dump.txt
```

Comment:
Please grep the codebase here if there are not references to `kind-fleet-agent-clusterscope `.

Reply:
There are only references in tests  and docs of kubernetes test package